sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -154,9 +158,10 @@ class FeatureFlagConfigurationSetting(
     )
     kind = "FeatureFlag"
 
-    def __init__(self, feature_id, enabled, filters=[], **kwargs):  # pylint: disable=dangerous-default-value
+    def __init__(self, feature_id, filters=[], **kwargs):  # pylint: disable=dangerous-default-value
         # type: (str, bool, Optional[List[Dict[str, Any]]], **Any) -> None
         super(FeatureFlagConfigurationSetting, self).__init__(**kwargs)
+        self.feature_id = feature_id.lstrip(self.key_prefix)
```

Comment:
Why?

Reply:
the feature_id (in the portal) does not have the prefix so this removes it incase the customer prefixes it with it (and takes care of it when deserializing).

Reply:
If user explicitly adds key_prefix, it's on their own. :)